cfme/tests/automate/test_simulation.py

```diff
@@ -86,3 +88,64 @@ def test_assert_failed_substitution(copy_class):
             request="Call_Instance",
             execute_methods=True,
         )
+
+
+@pytest.mark.tier(3)
+@pytest.mark.meta(automates=[1445089])
+def test_automate_simulation_result_has_hash_data(custom_instance):
+    """
+    The UI should display the result objects if the Simulation Result has
+    hash data.
+
+    Bugzilla:
+        1445089
+
+    Polarion:
+        assignee: ghubale
+        casecomponent: Automate
+        caseimportance: medium
+        initialEstimate: 1/6h
+        tags: automate
+        testSteps:
+            1. Create a Instance under /System/Request called ListUser, update it so that it points
+               to a ListUser Method
+            2. Create ListUser Method under /System/Request, paste the Attached Method
+            3. Run Simulation
+        expectedResults:
+            1.
+            2.
+            3. The UI should display the result objects
+    """
+    from . import user_list_hash_data
```

Comment:
what's the reason for having import statement here ?

Reply:
I imported ```user_list_hash_data``` here. Because it is only related to this test case and will not be used in any other test case. 
Please let me know your views @izapolsk

Reply:
@ganeshhubale, actually, according to pep8 all imports should be put in the head of file.
there are exceptional cases when we put imports inside some code when those cause some import issues.

Reply:
Thanks @izapolsk :+1: 
I didn't get pep8 error for this import. So kept as it is.
I will update as you suggested.